src/components/BannerRoutes.vue

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+import { defineComponent } from 'vue';
+
+export default defineComponent({
+  name: 'BannerRoutes',
+  props: {
+    routesCount: {
+      type: Number,
+      required: true,
+    },
+  },
+});
+</script>
+
+<template>
```

Comment:
This component is visually "same" as  `BannerImage` component. Could we reuse it again here?